packages/flutter/lib/src/widgets/localizations.dart

```diff
@@ -416,14 +416,38 @@ class Localizations extends StatefulWidget {
   /// true, in which case it returns null.
   static Locale? localeOf(BuildContext context, { bool nullOk = false }) {
     assert(context != null);
-    assert(nullOk != null);
     final _LocalizationsScope? scope = context.dependOnInheritedWidgetOfExactType<_LocalizationsScope>();
     if (nullOk && scope == null)
       return null;
-    assert(scope != null, 'a Localizations ancestor was not found');
+    assert(() {
+      if (scope == null) {
+        throw FlutterError(
+          'Localizations operation requested with a context that does not include Localizations.\n'
```

Comment:
..does not include a Localizations ancestor

Reply:
OK, I updated the wording.